docs/docsource/reference/introduction/supported_apis.rst

```diff
@@ -1,7 +1,7 @@
 Supported APIs
 ==============
 
-Supported APIs for Zenko are detailed here.
+Supported APIs for XDM   are detailed here.
```

Comment:
added a few spaces here

Reply:
Why? They'll just render out in the final.

Reply:
ok I will ignore these then, good to know.

Reply:
Okay... You talked me into making the change. I've pushed another commit... This one removes all the "XDM   " instances from running text, but leaves the incredibly brittle table spacing alone. Given the tools I have at my disposal (a motley assortment of bash pattern-matching tricks, sed, and the while...do loop) I am terrified of trying to fix the remaining "XDM   " instances. If you can provide sage bash counsel or some python-fu, I would be eager to learn.

Reply:
To state the problem explicitly, for the remaining "XDM   " instantly, I don't know how to make bash read in the remaining "XDM\ \ \ " instances (these are all in tables) and  A) figure which cell the "XDM   " is in, then B) replace with "XDM", then C) figure which "|" is the cell boundary (substitution text is also delimited by pipes, and some of these are in the tables) and D) inject the missing spaces in front of the cell boundary marker. 

This appears well and truly out of my wheelhouse.